### PR TITLE
LPS-144159 Add backend support for importing multiple entries

### PR DESCRIPTION
**Pull request by adolfopa**
These changes are backwards compatible. We'll add a feature flag when integrating the feature in the UI (via the management toolbar.)

### CODE REVIEW

**Comment by adolfopa** [comment]
This depends on LPS-144042, that hasn't been merged yet. See https://github.com/liferay-lima/liferay-portal/pull/2569.

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:stop

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java [hunk 1]
```diff
@@ -256,8 +255,16 @@ public String getTitle() throws PortalException {
...
 
-		Stream<String> stream = Arrays.stream(
-			infoItemLanguagesProvider.getAvailableLanguageIds(_model));
+		List<String> languageIds = new ArrayList<>();
+
+		for (Object model : _models) {
+			_intersect(
+				languageIds,
+				Arrays.asList(
+					infoItemLanguagesProvider.getAvailableLanguageIds(model)));
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java:247]: Can happens to be a no intersect languages for the different models?

**Review by AliciaGarciaGarcia — APPROVED** [review]
some questions but LGTM

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java:247]: The site default locale is always added to the set, so even if the intersection is empty, the returned set of languages will have at least one element.

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**